### PR TITLE
Feat: Admin Page

### PR DESCRIPTION
**Pull request by Setanna**

### CODE REVIEW

**Comment by Setanna** [comment]
[GUI - Admin](https://trello.com/c/MxGADdrT/24-gui-admin)

File: lib/pages/admin.dart [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  }
+
+  Widget locations() {
+    return DropdownSearch(
+      asyncItems: (String filter) async {
+        final locations = await api.fetchLocations();
+        return locations;
+      },
+      onChanged: (value) async {
+        debugPrint('Location: ${value!.id}');
```
Comment (MathiasDuus) [inline lib/pages/admin.dart:57]: Should be removed

File: lib/pages/admin.dart [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                                      alignment: Alignment.centerLeft,
+                                      child: Text(
+                                          "Telefon nummer: ${users.data?[index]['phone_number']}"),
+                                    ),
+                                    Align(
+                                      alignment: Alignment.centerLeft,
+                                      child: Text(
+                                          "Lokation: ${users.data?[index]['location']}"),
+                                    ),
+                                  ],
```
Comment (MathiasDuus) [inline lib/pages/admin.dart:131]: This is like the only place we use dansih in the app, so i think to keep the consistency, it should be englisg aswell

**Review by MathiasDuus — CHANGES_REQUESTED** [review]
One last small thing, can you rename the file to end with "_page.dart" to keep consistency.

Otherwise it looks good

File: lib/pages/admin.dart (hunk 1 above)
Comment (Setanna) [reply lib/pages/admin.dart]: Has been removed

File: lib/pages/admin.dart (hunk 2 above)
Comment (Setanna) [reply lib/pages/admin.dart]: Changed to english

**Review by MathiasDuus — APPROVED** [review]

**Pull request merged**